projects/Mallard/src/components/front/items/text-block.tsx

```diff
@@ -56,6 +60,23 @@ const useTextBlockStyles = (textBlockAppearance: TextBlockAppearance) => {
     }
 }
 
+const getFontSize = ({ layout, story }: ItemSizes) => {
+    if (layout === PageLayoutSizes.tablet) {
+        if (story.width >= 3) {
+            if (story.height >= 3) return 1.75
+            if (story.height >= 2) return 1.25
+        }
+        if (story.width >= 2) {
+            if (story.height >= 3) return 1.5
+            // 2x1s are text only so we bump them up
```

Comment:
Could we have a `textOnly` only property on `ItemSizes` so that we don't need to couple are layout decisions to the current implementation of certain shapes?

Reply:
yup! thinking of doing its own component (LargeTextNoImageItem or something like that)?

Reply:
👌 sounds good!